pkgs/applications/editors/netbeans/default.nix

```diff
@@ -1,9 +1,9 @@
 { stdenv, fetchurl, makeWrapper, makeDesktopItem, which, unzip, libicns, imagemagick
-, jdk, perl, python
+, jdk11, perl, python
```

Comment:
Maybe keep this `jdk` and pass `jdk = jdk11` in the callPackage attrset? This would make it easier and more predictable for others to override the JDK.